scripts/common.sh

```diff
@@ -26,6 +26,11 @@ set -e
 # changes directory. The paths returned by `caller` are relative to it.
 _initial_work_dir=$(pwd)
 
+if [ -z ${CIRCLECI+x} ]
+then
+    CIRCLECI=0
+fi
+
 if [ "$CIRCLECI" ]
```

Comment:
Why not just this?
```suggestion
if [[ ${CIRCLECI:-} != "" ]]
```

I.e. Use put the check directly wherever we need this variable. Just setting it to `0` could be interpreted in some places as true.

Also the more modern `if` syntax and `${var:-default}`, to give it an empty string as a default value when the variable is undefined.

Reply:
Thats much better!